solutions/USACO jan21-dancemooves.mdx

```diff
@@ -0,0 +1,105 @@
+---
+id: usaco-1086
+source: USACO Silver 2021 January
```

Comment:
wrong order LOL

Reply:
seems to work fine for other solutions it seems
![image](https://user-images.githubusercontent.com/43394822/108612685-953a5b00-73b0-11eb-92c3-f64fd85dda5f.png)

Reply:
oops mb